### ISSUE TITLE
Latest alpha on npm?

### ISSUE THREAD

**Issue by taikn** [nkbt/react-collapse#239]
Hey, any chance latest alpha can be made available through npm?

**Comment by nkbt** [comment]
yeah you can install `react-collapse@next`, or use `@nkbt/react-collapse`

**Comment by nkbt** [comment]
ah right... I decided it would be safer and more convenient to do releases in a `@nkbt` namespace, so we do not need to mess with locked versions in our projects
